2.6.md

```diff
@@ -802,6 +802,11 @@ There are also several subordinate objects that provide detailed data to potenti
     <td>object</td>
     <td>A Regs object (Section 3.2.3) that specifies any industry, legal, or governmental regulations in force for this request.</td>
   </tr>
+  <tr>
+    <td><code>acat</code></td>
+    <td>string array</td>
+    <td>Allow list of category of apps permitted for the offered ad opportunity</td>
```

Comment:
Is there a specific taxonomy in which acat would be expressed?

Reply:
Yes - as with **bcat**, The taxonomy to be used is defined by the cattax field. If no
cattax field is supplied IAB Content Category Taxonomy 1.0 is
assumed.

Reply:
Please update language to reflect inverse of bcat